.splunk/conf/outputs.conf

```diff
@@ -0,0 +1,10 @@
+[indexAndForward]
+index = false
+
+[tcpout]
+defaultGroup = nginx
+
+[tcpout:nginx]
+server = dps-splunk:9997
+
+[tcpout-server://dps-splunk:9997]
```

Comment:
Recommend setting this for local dev, we'll handle this in OpenShift differently

Reply:
so dps-splunk is the docker hostname which makes this running on docker, do you suggest we go localhost, but then when we do docker-compose up, we need to change this value ?

Reply:
oh - my bad.  Stick with the docker DNS entry.